config/default/field.field.node.campground_directory_record.field_map_upload.yml

```diff
@@ -16,7 +16,7 @@ id: node.campground_directory_record.field_map_upload
 field_name: field_map_upload
 entity_type: node
 bundle: campground_directory_record
-label: 'Map download'
+label: 'Map upload'
```

Comment:
Do we need this?

Reply:
yes https://github.com/ytgov/yukon-ca/pull/51/files#diff-0363838414b88103beed6da25f676cedfb277b0b0df3dd21b27ffa3aadfb8c5eR19

Reply:
reverted